docs/spec_draft.md

```diff
@@ -2215,6 +2216,33 @@ implementation-defined value.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.replica_id
+
+### Semantics
+
+Returns the unique id of the replica in the interval [0, N), where N is the
```

Comment:
can we link the parallel execution semantics for users to understand what a replica is

Reply:
IMO we can skip that given that we have a section dedicated to it in the beginning. This is the same treatment we did to token.